Feature/job accept

shyanrafer opened the pull request.

shyanrafer committed:
Working out the pricing page, added a get route to users, created front end file for job signup.

shyanrafer committed:
Spent some time trying to work out job status change. Couldn't resolve.

ClassJa reviewed (changes requested):
The buttons are currently rendering properly, let's check with the rest of the team and see if that is true on their machines as well and handle this pr then as well.

shyanrafer commented:
Earlier i had switched to main, pulled, then ran things and got what i mentioned in the group chat. I switched back to feature and then back to main and now things are working.

shyanrafer committed:
Attempted to unify style for pricing and available jobs page.
